tests/acceptance/gcp-auth.bat

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bats
+
+#load _helpers
+#
+#SKIP_TEARDOWN=true
+VAULT_ADDR='http://127.0.0.1:8200'
+VAULT_IMAGE="${VAULT_IMAGE:-hashicorp/vault:1.9.0-rc1}"
+
+if [[ -z SERVICE_ACCOUNT_ID ]]
+then
+    echo "SERVICE_ACCOUNT_ID env is not set. Exiting.."
+    exit 1
+fi
+
+if [[ -z PATH_TO_CREDS ]]
+then
+    echo "PATH_TO_CREDS env is not set. Exiting.."
+    exit 1
+fi
+
+export SETUP_TEARDOWN_OUTFILE=/tmp/output.log
+
+setup(){
+    { # Braces used to redirect all setup logs.
+    # 1. Configure Vault.
+    VAULT_TOKEN='root'
+    DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+    docker pull ${VAULT_IMAGE?}
+
+    docker run \
+      --name=vault \
+      --hostname=vault \
+      -p 8200:8200 \
+      -e VAULT_DEV_ROOT_TOKEN_ID="root" \
+      -e VAULT_ADDR="http://localhost:8200" \
+      -e VAULT_DEV_LISTEN_ADDRESS="0.0.0.0:8200" \
+      --privileged \
+      --detach ${VAULT_IMAGE?}
+
+    # Replace with a check
```

Comment:
If you want to replace this with a check you could use something like:
```
    echo -n "waiting for vault"
    while ! vault status >/dev/null 2>&1; do sleep 1; echo -n .; done; echo
```

Reply:
Thanks! Added the check in setup